src/index.html

```diff
@@ -18,8 +18,8 @@
 }
 
 render();
-if (module.hot) {
-  module.hot.accept(render);
-}
+
+module.hot = false;
```

Comment:
We should be able to just delete this line. Might want to do a find for `hot` in `/src` to see if we use it anywhere, then delete it.